duffel_api/api/booking/offer_requests.py

```diff
@@ -15,7 +15,9 @@ def __init__(self, **kwargs):
 
     def get(self, id_):
         """GET /air/offer_requests/:id"""
-        return OfferRequest(self.do_get("{}/{}".format(self._url, id_))["data"])
+        return OfferRequest.from_json(
+            self.do_get("{}/{}".format(self._url, id_))["data"]
```

Comment:
Since [we're probably going to move to f-strings](https://www.notion.so/duffel/Python-SDK-db0afb7cafe74c05ba5c2dc9fcbc722e#4f2e1535230044be8b3fba69008c7a4f) over the codebase, may as well start here:

```python
             self.do_get(f"{self._url}/{id_}"))["data"]
```

Reply:
I'll leave the f-strings for another PR!